.gitignore

```diff
@@ -7,3 +7,7 @@ dist/
 *.sw[pon]
 .coverage
 .tox
+
+# IntelliJ project and module files
+.idea
+*.iml
```

Comment:
Please revert this unrelated change.

Reply:
Those are IntelliJ project files added to gitignore. Why is this an issue?

Reply:
I don't see any reason to add IDE-specific files to `.gitignore`.

Reply:
Ok, you’re probably right. I will add those to my global gitignore and revert. Thanks for your feedback!